src/uniprotkb/components/landing-page/StatisticsChart.tsx

```diff
@@ -0,0 +1,236 @@
+    .style('stroke-width', '2px');
+
+  slice.exit().remove();
+
+  /* ------- TEXT LABELS -------*/
+
+  const text = g
+    .select('.labels')
+    .selectAll<SVGTextElement, d3.PieArcDatum<StatisticsItem>>('text')
+    .data(pie(data));
+
+  const midAngle = (d: d3.PieArcDatum<StatisticsItem>) =>
+    d.startAngle + (d.endAngle - d.startAngle) / 2;
+
+  text
+    .enter()
+    .append('text')
+    .attr('dy', '.35em')
+    .style('pointer-events', 'auto')
+    .style('cursor', 'pointer')
+    .style('fill', '#014371')
+    .style('font-weight', 'bold')
+    .on('click', (d) => {
+      history.push(
+        stringifyUrl(LocationToPath[Location.UniProtKBResults], {
+          query: `taxonomy_name: ${d.data.name}`,
+        })
+      );
+    })
+    .text((d) => d.data.name)
+    .merge(text)
+    .transition()
+    .duration(1000)
+    .attrTween('transform', (d) => {
+      const interpolate = d3.interpolate(current, d);
+      current = interpolate(0);
+      return (t) => {
+        const d2 = interpolate(t);
+        const pos = outerArc.centroid(d2);
+        pos[0] = radius * (midAngle(d2) < Math.PI ? 1 : -1);
+        return `translate(${pos})`;
+      };
+    })
+    .styleTween('text-anchor', (d) => {
+      const interpolate = d3.interpolate(current, d);
+      current = interpolate(0);
+      return (t) => {
+        const d2 = interpolate(t);
+        return midAngle(d2) < Math.PI ? 'start' : 'end';
+      };
+    });
+
+  text.exit().remove();
+
+  /* ------- SLICE TO TEXT POLYLINES -------*/
+
+  const polyline = g
+    .select('.lines')
+    .selectAll<SVGPathElement, d3.PieArcDatum<StatisticsItem>>('polyline')
+    .data(pie(data), key);
+
+  polyline
+    .join('polyline')
+    .transition()
+    .duration(1000)
+    .attrTween('points', (d) => {
+      const interpolate = d3.interpolate(current, d);
+      current = interpolate(0);
+      return (t) => {
+        const d2 = interpolate(t);
+        const pos = outerArc.centroid(d2);
+        pos[0] = radius * 0.95 * (midAngle(d2) < Math.PI ? 1 : -1);
+        const pointsArray = [midArc.centroid(d2), outerArc.centroid(d2), pos];
+        return pointsArray.map((point) => point.join(',')).join(' ');
+      };
+    })
+    .style('fill', 'none')
+    .style('opacity', 0.3)
+    .style('stroke', 'black')
+    .style('stroke-width', '2px');
+
+  polyline.exit().remove();
+};
+
+const StatisticsChart = ({ releaseNumber }: { releaseNumber?: string }) => {
+  const history: History = useHistory();
+  const svgRef = useRef<SVGSVGElement>(null);
+
+  const reviewedStats = useDataApi<StatisticsPayload>(
+    releaseNumber &&
+      `${API_PREFIX}/statistics/releases/${releaseNumber}/reviewed`
+  );
+
+  const unreviewedStats = useDataApi<StatisticsPayload>(
+    releaseNumber &&
+      `${API_PREFIX}/statistics/releases/${releaseNumber}/unreviewed`
+  );
+
+  useEffect(() => {
+    if (
+      svgRef.current &&
+      !reviewedStats.loading &&
```

Comment:
Any specific reason you're checking the loading flag only for reviewedStats but not for unreviewedStats?